packages/twenty-server/src/workspace/messaging/services/fetch-by-batch.service.ts

```diff
@@ -0,0 +1,133 @@
+import { Injectable } from '@nestjs/common';
+
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+
+import { BatchQueries } from 'src/workspace/calendar-and-messaging/types/batch-queries';
+import { GmailMessageParsedResponse } from 'src/workspace/messaging/types/gmail-message-parsed-response';
+
+@Injectable()
+export class FetchByBatchesService {
+  private readonly httpService: AxiosInstance;
+
+  constructor() {
```

Comment:
Instead you should inject the httpService in the constructor, and instantiate a new `HttpModule` in the module with the `baseUrl`